src/guide/get-started/operate-iroha-via-cli.md

```diff
@@ -0,0 +1,359 @@
+# Operate Iroha via CLI
+
+In this tutorial, we will set up and configure the Iroha client CLI, and perform some basic actions on the Iroha blockchain with it.
+
+You can learn how to:
+  - register [domains](#_3-register-a-domain) and [accounts](#_4-register-an-account)
+  - [mint](#_6-register-and-mint-assets), [transfer](#_7-transfer-assets) and [burn](#_8-burn-assets) assets
+  - [visualize outputs](#_9-visualize-outputs)
+
+## 1. Setup Iroha 2 Client
+
+Ensure you [installed Iroha](./install-iroha.md) and [brought up the test network](./launch-iroha.md). 
+
+Create a fresh directory and copy the `client.toml` configuration file there:
+
+```bash
+$ cp path_to_iroha_repo/defaults/client.toml path_to_created_directory/
+```
+
+## 2. Configure Iroha 2 Client
+
+Navigate to the directory with the configuration file in your terminal.
+
+Run the client CLI:
+
+```bash
+$ iroha
+```
+
+::: details Expand to see the expected output
+
+```bash
+Iroha CLI Client lets you interact with Iroha Peers Web API without direct network usage
+
+Usage: iroha [OPTIONS] <COMMAND>
+
+Commands:
+  domain   The subcommand related to domains
+  account  The subcommand related to accounts
+  asset    The subcommand related to assets
+  peer     The subcommand related to p2p networking
+  events   The subcommand related to event streaming
+  wasm     The subcommand related to Wasm
+  blocks   The subcommand related to block streaming
+  json     The subcommand related to multi-instructions as Json or Json5
+  help     Print this message or the help of the given subcommand(s)
+
+Options:
+  -c, --config <PATH>  Path to the configuration file [default: client.toml]
+  -v, --verbose        More verbose output
+  -h, --help           Print help
+  -V, --version        Print version
+
+```
+
+:::
+
+By default, the Iroha client searches for its configuration in the `client.toml` file located in the current working directory. We already copied it there, so we're all set.
+
+::: tip
+
+To run any of the Iroha client commands with some other configuration file, use the following syntax:
+
+```bash
+$ iroha --config path/to/client.toml <COMMAND>
+```
+
+This is not _persistent configuration_: each time you run `iroha`, you must add the `--config path/to/client.toml` command-line argument, unless you put the `client.toml` config file in your working directory.
+
+:::
+
+The account specified in the `[account]` section of `client.toml` is preregistered in the blockchain. Only it can interact with the blockchain for now. If you change the keys or the domain of the account in the config file, you should ensure they are preregistered in the blockchain too.
```

Comment:
Would be helpful to clarify that this is configured in the default `genesis.json`

Reply:
fixed in commit fec265d9a98b1d356133521d2320837db466d62f